### PR TITLE
Support newer Python and Robot Framework versions

### PR DESCRIPTION
**Pull request by vereszol**
Time has passed, this tool should be updated.

### CODE REVIEW

File: src/crl/rfcli/rfcli.py
Comment (OneTorcha) [inline src/crl/rfcli/rfcli.py:1]: Can you add at the same the change to execute()

        for key, value in self.new_environment_variables.items():
            newpaths = value.split(os.pathsep)
            for path in newpaths:
                if not any(fnmatch.fnmatch(p, path) for p in sys.path):
                    sys.path.append(path)
        return run_cli(self.commandline[1:], exit=False)  # pylint: disable=unsubscriptable-object

this pythonpathsetter add_path() function is gone.

for path in newpaths:
                pythonpathsetter.add_path(path, True)

https://github.com/robotframework/robotframework/blob/master/src/robot/pythonpathsetter.py

Comment (vereszol) [reply src/crl/rfcli/rfcli.py:1]: Yes, that shall be added together with support for newer Robot Framework versions, as I pinned it down to <4.0 in 8fcb9b9 to get tox tests passing.

Comment (vereszol) [reply src/crl/rfcli/rfcli.py:1]: Done in 094f2a57c39e9

**Review by OneTorcha — APPROVED** [review]
Looks perfect.

**Comment by theodore86** [comment]
bump

**Pull request merged**